content/en/docs/ops/configuration/traffic-management/network-topologies/index.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Configuring Gateway Network Topology (Development)
+description: How to configure gateway network topology (Development).
+weight: 60
+keywords: [traffic-management,ingress,gateway]
+---
+
+## (Development) Configuring network topologies
+
+Istio provides the ability to manage settings like [X-Forwarded-For](https://tools.ietf.org/html/rfc7239)
+([XFF](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for)
+and [X-Forwarded-Client-Cert](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-client-cert)
+(XFCC),  which are dependent on how the gateway workloads are deployed. This is currently an in-development feature.
+
+Many users choose to deploy Istio ingress gateways in using various network topologies
+(e.g. behind Cloud Load Balancers, a self-managed Load Balancer or directly expose the
+Istio ingress gateway to the Internet). As such, these topologies require different ingress gateway configurations for
+transporting correct client attributes like IP addresses and certificates to the workloads running in the cluster.
+
+Configuration of XFF and XFCC headers can be configured by using `MeshConfig` during Istio
+*installation* or by adding a pod annotation.
```

Comment:
is mesh config only for global change?

Reply:
Yes, "mesh (or proxy) config" is global for all Gateway workloads and you can use annotations to customize per GW workload. We can clarify it in the doc.